Sources/Prometheus/MetricTypes/Summary.swift

```diff
@@ -1,8 +1,11 @@
+import NIOConcurrencyHelpers
+
 /// Default quantiles used by Summaries
 public var defaultQuantiles = [0.01, 0.05, 0.5, 0.9, 0.95, 0.99, 0.999]
```

Comment:
terrifying public global var! Please make it a `let` or otherwise we'd need to protect potential concurrent modifications. (or move it into a class and protect it there)

Reply:
+1